### PR TITLE
Add initial vividus dsl syntax support

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: language-configuration.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    [
+      "(",
+      ")"
+    ],
+    [
+      "[",
+      "]"
+    ],
+    [
+      "{",
```
Comment (ikalinin1) [inline language-configuration.json:12]: "${", "}" ?

File: package.json [hunk 2]
```diff
@@ -14,6 +14,35 @@
...
+            {
+                "id": "vividus-dsl",
+                "aliases": [
+                    "bdd",
+                    "jbehave",
+                    "dsl"
+                ],
+                "extensions": [
+                    ".story",
+                    ".steps"
```
Comment (ikalinin1) [inline package.json:27]: table ?

File: syntaxes/vividus.tmLanguage.json [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+                "1": {
+                    "name": "vividus.meta.key"
+                },
+                "2": {
+                    "name": "vividus.meta.value"
+                }
+            }
+        },
+        "step": {
+            "match": "^(Given(?!Stories)|When|Then|And)(.*)",
```
Comment (ikalinin1) [inline syntaxes/vividus.tmLanguage.json:86]: .* -> .+ ?

File: language-configuration.json (hunk 1 above)
Comment (uarlouski) [reply language-configuration.json:12]: it's not a pair

File: package.json (hunk 2 above)
Comment (uarlouski) [reply package.json:27]: there is not highlighting for table

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

**Pull request merged**